.goreleaser.yml

```diff
@@ -41,6 +41,10 @@ builds:
     id: windows
     goos: [windows]
     goarch: ["386", amd64, arm64]
+    hooks: # https://goreleaser.com/customization/builds/#build-hooks
```

Comment:
We should probably just close off this PR for now until we have a plan to move forward.